src/networks.json

```diff
@@ -397,72 +226,6 @@
     "start": 5076543,
     "logo": "ipfs://Qmd7dKnNwHRZ4HRCbCXUbkNV7gP28fGqPdjbHtjRtT9sQF"
   },
-  "69": {
-    "key": "69",
-    "name": "Optimism Kovan",
-    "chainId": 69,
-    "network": "Optimism testnet",
-    "testnet": true,
-    "multicall": "0x28e9a2329aa6b675ca251a2ccaa7fb029c1e9bfb",
-    "rpc": [
-      "https://opt-kovan.g.alchemy.com/v2/JzmIL4Q3jBj7it2duxLFeuCa9Wobmm7D"
-    ],
-    "explorer": {
-      "url": "https://kovan-optimistic.etherscan.io"
-    },
-    "start": 882942,
-    "logo": "ipfs://QmfF4kwhGL8QosUXvgq2KWCmavhKBvwD6kbhs7L4p5ZAWb"
-  },
-  "70": {
-    "key": "70",
-    "name": "Hoo Smart Chain Mainnet",
-    "shortName": "hsc",
-    "chainId": 70,
-    "network": "Mainnet",
-    "multicall": "0xd4b794b89baccb70ef851830099bee4d69f19ebc",
-    "rpc": [
-      "https://http-mainnet2.hoosmartchain.com"
-    ],
-    "ws": [
-      "wss://ws-mainnet2.hoosmartchain.com"
-    ],
-    "explorer": {
-      "url": "https://hooscan.com"
-    },
-    "start": 404118,
-    "logo": "ipfs://QmNhFCVw5GDsu86sGchoRNvQjcr5GL79UJQ3xCHzdFbZYk"
-  },
-  "74": {
-    "key": "74",
-    "name": "IDChain",
-    "shortName": "IDChain",
-    "chainId": 74,
-    "network": "mainnet",
-    "multicall": "0x41d289c966D51342A515A19dE9c27d16079c94E0",
-    "rpc": [
-      "https://idchain.one/archive/rpc/",
-      "https://idchain.songadao.org/rpc"
-    ],
-    "explorer": {
-      "url": "https://explorer.idchain.one"
-    },
-    "start": 10780012,
-    "logo": "ipfs://QmXAKaNsyv6ctuRenYRJuZ1V4kn1eFwkUtjrjzX6jiKTqe"
-  },
-  "75": {
-    "key": "75",
-    "name": "Decimal",
-    "shortName": "mainnet",
-    "chainId": 75,
-    "network": "mainnet",
-    "multicall": "0x949d1A0757803C51F2EfFFEb5472C861A898B8E8",
-    "rpc": [],
-    "explorer": {
-      "url": "https://explorer.decimalchain.com"
-    },
-    "start": 16031065,
-    "logo": "ipfs://bafkreihkdhbce5rkogl63xegaarlirjrvbfarxbtbf5mqme3s5grvbjyxm"
-  },
```

Comment:
This is added very recently, https://github.com/snapshot-labs/snapshot.js/pull/894 better to keep this